package.json

```diff
@@ -8,7 +8,7 @@
     "build": "npm-run-all --parallel build:css build:html build:js",
     "build:html": "ncp ./src/index.html ./dist/index.html && ncp ./src/assets/svg-filters.svg ./dist/svg-filters.svg",
     "build:css": "node scripts/postcss.js",
-    "build:js": "uglifyjs --compress --output ./dist/index.js -- ./src/assets/a11y-viewer.js",
+    "build:js": "ncp ./src/assets/a11y-viewer.js ./dist/index.js",
```

Comment:
i would uglify, if you need non uglified js in development. Create a seperate issue for development/production build